### PR TITLE
Work around bug 1610661

### PR DESCRIPTION
**Pull request by liangxia**
To replace #168 

@weherdh  @chao007  @xingxingxia

Jenkins log, Runner-v3/40224

### CODE REVIEW

**Comment by weherdh** [comment]
LGTM

File: features/step_definitions/pv.rb [hunk 1]
```diff
@@ -156,22 +156,21 @@
...
   @result = BushSlicer::PersistentVolume.create(by: admin, spec: pv_hash)
 
   if @result[:success]
-    cache_resources *@result[:resource]
-
-    # register mandatory clean-up
-    _pv = @result[:resource]
     _admin = admin
     teardown_add {
-      @result = _pv.delete(by: _admin)
```
Comment (weherdh) [inline features/step_definitions/pv.rb]: Sorry, I met this error: https://openshift-qe-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Runner-v3/40227/console, and not sure why. Does `delete_deps` here work?

Comment (liangxia) [reply features/step_definitions/pv.rb]: @weherdh Fixed. See Runner-v3/40240
Also please update your comments to remove the internal URL.

**Comment by liangxia** [comment]
@pruan-rht @akostadinov

File: features/step_definitions/pv.rb [hunk 2]
```diff
@@ -164,14 +165,15 @@
     cache_resources *@result[:resource]
 
     # register mandatory clean-up
-    _pv = @result[:resource]
     _admin = admin
     teardown_add {
-      @result = _pv.delete(by: _admin)
```
Comment (akostadinov) [inline features/step_definitions/pv.rb]: actually, you don't need to convert to `cli_exec`. You could just make this `_pv.delete(by: _admin, wait: "false")`.

Comment (akostadinov) [reply features/step_definitions/pv.rb]: wait

**Comment by akostadinov** [comment]
Superseded by #172 

Sorry, I figured fixing one layer deeper makes even more sense.

**Pull request closed without merging**